examples/kafka-streams-examples/docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+---
+version: '2'
+services:
+  zookeeper:
+    image: confluentinc/cp-zookeeper:3.2.0
```

Comment:
@arrawatia: So, what should I do (not fully clear to me yet).  Change from `:3.2.0` to `:latest` for both examples and tests?